### DISCUSSION TITLE
[Esame 24 Aprile 2014] Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#252] [Esami - Bertini @ Matematica]
![Screenshot 2023-11-14 165819](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/37d4351d-d982-493c-8386-05ffb5103547)

**Comment by CasuFrost** [comment]
Punti (i), (ii), (iii) e (iv) : 
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/97389762/dfff38e0-3984-4b94-8d76-ab0495079f76)
